model/textparse/promparse.go

```diff
@@ -66,6 +66,9 @@ const (
 	tEqual
 	tTimestamp
 	tValue
+
+	quantileString = "quantile"
+	leString = "le"
```

Comment:
Don't we have these somethere as constants? (Or should we make them?)

Reply:
They are already constants here, sorry.

But here is where they should come from:

```Go
import "github.com/prometheus/common/model"

model.QuantileLabel
model.BucketLabel
```